### ISSUE TITLE
[/collect/person/] Christopher Wix bio submission

### ISSUE THREAD

**Issue by ntbot** [newtheatre/history-project#524]
End user submitted issue from page: [/collect/person/](http://alpha.history.newtheatre.org.uk/collect/person/)
---

# 'Submit an almni bio' form submission

Field | Data
----- | ----
Name | Christopher Wix
Grad Year | 1995
Course | History

## Bio1 (Time at theatre)

Doing the tech on Westside Story whilst at the same time being in the play!!!

## Bio2 (Post-graduation)

International Tax

Field | Data
------| ----
Checked careers | 
Other careers | 

## Shows

The Dream Play Autumn 92
Lysistrata 94
Westside Story
Macbeth
A couple of others!!!

## Committees

Lighting Manager 93/94

## Contact Preferences

Are we allowed to facilitate contact to this alumnus? **Yes**

## Attempted File Generation

```
---
title: Christopher Wix
course:
  - History
graduated: 1995
contact_allowed: true
career:


links: *fill me out

award: *fill me out

---

Doing the tech on Westside Story whilst at the same time being in the play!!!

International Tax

```

**

**Comment by nhyty** [comment]
Closed in edab932

**Comment by wjdp** [comment]
> Closed in edab932

That don't exist!
